README.md

```diff
@@ -117,6 +117,7 @@ more "reports" on them.
 ```ruby
 api.check.create('applicant_id', type: 'express', reports: [{ name: 'identity' }])
 api.check.find('applicant_id', 'check_id')
+api.check.find_by_url(params[:payload][:object][:href])
```

Comment:
This example does make sense outside of a rails context (`params` being controller params). Perhaps we could pass a partial URL instead for this example? e.g. `api.check.find_by_url('applicants/899fcd0c-0078-4181-93eb-db9f7f297d3a/checks/5e97140e-fe9c-4b6d-bb31-8a1ccbe2faa9')`

Reply:
Ah yeah, makes sense